src/features/domain-settings/DomainSettings.constants.ts

```diff
@@ -0,0 +1,110 @@
+import { ConfirmActionType } from './DomainSettings.types';
+
+/**************************
+ * COMPLETE_STEP_LABEL_TEXT
+ **************************/
```

Comment:
nit: for doc comments, write them out in plain English. The variable name is already down below so the doc comment should just give a little description of what you're looking at. Also worth noting, if you put the doc comment above the exported function, your IDE will show you the doc comment whenever you use the function. E.g.
```
/**
 * Adds two numbers
 */
const add = ...
```
Will show "add -> Adds two numbers" when you hover over the function. Very handy 😊